manifests/b/BlueStack/BlueStacks/5.5.101.1001/BlueStack.BlueStacks.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: BlueStack.BlueStacks
+PackageVersion: 5.5.101.1001
+MinimumOSVersion: 10.0.0.0
+InstallerSwitches:
+  Silent: -s
+  SilentWithProgress: -s
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://cdn3.bluestacks.com/downloads/windows/nxt/5.5.101.1001/65a19fda4a914e2ea8f266863149b658/FullInstaller/x64/BlueStacksFullInstaller_5.5.101.1001_amd64_native.exe?filename=BlueStacksInstaller_5.5.101.1001_amd64_native_e0175934c4a0a56f1db683585ffd1bcf.exe

```

This might be a better url to use? 
```suggestion
  InstallerUrl: https://cdn3.bluestacks.com/downloads/windows/nxt/5.5.101.1001/65a19fda4a914e2ea8f266863149b658/FullInstaller/x64/BlueStacksFullInstaller_5.5.101.1001_amd64_native.exe
```